tests/unittests/config/test_apt_configure_sources_list_v1.py

```diff
@@ -120,12 +148,12 @@ def apt_source_list(self, distro, mirror, tmpdir, mirrorcheck=None):
     def test_apt_v1_source_list_by_distro(self, distro, mirror, tmpdir):
         """Test rendering of a source.list from template for each distro"""
         mycloud = get_cloud(distro)
-        tmpl_file = f"/etc/cloud/templates/sources.list.{distro}.tmpl"
-        util.write_file(tmpl_file, EXAMPLE_TMPL)
+        tmpl_file = f"/etc/cloud/templates/sources.list.{distro}.deb822.tmpl"
```

Comment:
The old test is still a valid test and valid behavior, correct? Should we have a version of the test that uses the feature flag and one that doesn't, rather than moving to a version that only does?

The same concern applies to all tests modified/removed in this file and the v3 version.

Reply:
The deb822 feature is being tested in other unittests and that code path is the same regardless of v1 vs v3 vs deb822 or not. As long as we have a unit test that asserts deb822 vs non-deb822 feature finds the correct deb822 template file we know this will work for all unitttests. Addiing a matrix to each unittest to test v3 vs v1 finding the deb822 template won't produce a signal beyond "deb822 feature flag can't find the right template file".  But, I did add `deb822_feature_aware` unittest names for those tests that are specifically validating deb822.tmpl functionality.